tests/Utilities.Test.UnitTests/test.runsettings

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+	<MSTest>
+		<Parallelize>
+			<Workers>8</Workers>
```

Comment:
is this something we want to be able to set (in build pipeline or something else) runsettings is also config. But I mean all config stuff in the same file

Reply:
I think this is a great idea. Split things like these into user settings and build settings.
Create #25 to track this